controllers/cloudflare_api.go

```diff
@@ -2,69 +2,45 @@ package controllers
 
 import (
 	"bytes"
+	"context"
 	"crypto/rand"
 	"encoding/base64"
 	"encoding/json"
 	"fmt"
 	"net/http"
 	"net/url"
 
+	"github.com/cloudflare/cloudflare-go"
 	"github.com/go-logr/logr"
 )
 
-// CLOUDFLARE_ENDPOINT is the Cloudflare API base URL from https://api.cloudflare.com/#getting-started-endpoints.
-const CLOUDFLARE_ENDPOINT = "https://api.cloudflare.com/client/v4/"
-
 // TXT_PREFIX is the prefix added to TXT records for whom the corresponding DNS records are managed by the operator.
 const TXT_PREFIX = "_managed."
```

Comment:
I do not think this is needed anymore since the TXT is being added on the CNAME itself, right?